SFML test fails on NetBSD

The test `tests/niminaction/Chapter8/sfml/sfml_test.nim` fails on netBSD with the following output:

```
Category: niminaction
Name: tests/niminaction/Chapter8/sfml/sfml_test.nim C++ --nilseqs:on
Action: compile
Result: reNimcCrash
-------- Expected -------

--------- Given  --------
$ /home/euant/Nim/bin/nim cpp --hints:on -d:testing --nimblePath:tests/deps --ni                                                                                           lseqs:on --nimCache:nimcache/tests/niminaction/Chapter8/sfml/sfml_test.nim_d8de4                                                                                           4f831a39fe22ef81cd6a2f25664 --stdout '--hint[Path]:off' '--hint[Processing]:off'                                                                                            tests/niminaction/Chapter8/sfml/sfml_test.nim
Hint: used config file '/home/euant/Nim/config/nim.cfg' [Conf]
Hint: used config file '/home/euant/Nim/config/config.nims' [Conf]
Hint: used config file '/home/euant/Nim/tests/config.nims' [Conf]
CC: stdlib_system.nim
CC: stdlib_posix.nim
CC: stdlib_times.nim
CC: stdlib_os.nim
CC: sfml_test.nim
/home/euant/Nim/nimcache/tests/niminaction/Chapter8/sfml/sfml_test.nim_d8de44f83                                                                                           1a39fe22ef81cd6a2f25664/@msfml_test.nim.cpp:10:10: fatal error: SFML/Graphics.hp                                                                                           p: No such file or directory
 #include <SFML/Graphics.hpp>
          ^~~~~~~~~~~~~~~~~~~
compilation terminated.
Error: execution of an external compiler program 'g++ -c -std=gnu++14 -funsigned                                                                                           -char  -w -w -fpermissive   -I/home/euant/Nim/lib -I/home/euant/Nim/tests/nimina                                                                                           ction/Chapter8/sfml -o /home/euant/Nim/nimcache/tests/niminaction/Chapter8/sfml/                                                                                           sfml_test.nim_d8de44f831a39fe22ef81cd6a2f25664/@msfml_test.nim.cpp.o /home/euant                                                                                           /Nim/nimcache/tests/niminaction/Chapter8/sfml/sfml_test.nim_d8de44f831a39fe22ef8                                                                                           1cd6a2f25664/@msfml_test.nim.cpp' failed with exit code: 1
```

Related to #14033 and #12141

--- a/tests/niminaction/Chapter8/sdl/sdl.nim
+++ b/tests/niminaction/Chapter8/sdl/sdl.nim
@@ -1,6 +1,6 @@
 when defined(Windows):
   const libName* = "SDL2.dll"
-elif defined(Linux) or defined(freebsd):
+elif defined(Linux) or defined(freebsd) or defined(netbsd):
   const libName* = "libSDL2.so"
 elif defined(MacOsX):
   const libName* = "libSDL2.dylib"

--- a/tests/niminaction/Chapter8/sfml/sfml_test.nim
+++ b/tests/niminaction/Chapter8/sfml/sfml_test.nim
@@ -3,6 +3,7 @@ action: compile
 disabled: "windows"
 disabled: "freebsd"
 disabled: "openbsd"
+disabled: "netbsd"
 """
 
 import sfml, os

